### PR TITLE
Add Indexer shutdown command

### PR DESCRIPTION
**Pull request by trevorbernard**

### CODE REVIEW

File: test [hunk 1]
```diff
@@ -79,27 +81,30 @@ idxr server config 2>&1 |
 # Indexer server cli subcommand works
 idxr server cli --help > /dev/null
 
+# Temporary hack - create startup-blocks direct if it doesn't exist
```
Comment (jenr24G) [inline test:84]: can we put this in `${TMPDIR}/startup`?

File: test [hunk 2]
```diff
@@ -79,27 +81,30 @@ idxr server config 2>&1 |
...
+mkdir $HOME/.mina-indexer/startup-blocks || true
```
Comment (jenr24G) [inline test:85]: we shouldn't need this since the tests override this to $BLOCKS_DIR

File: test [hunk 3]
```diff
@@ -79,27 +81,30 @@ idxr server config 2>&1 |
...
 server cli \
-    --startup-dir $TMP/blocks \
-    --watch-dir $TMP/blocks \
+    --startup-dir $BLOCKS_DIR \
+    --watch-dir $BLOCKS_DIR \
     --initial-ledger tests/ledgers/mainnet-genesis.json \
     --is-genesis-ledger \
     --root-hash 3NKeMoncuHab5ScarV5ViyF16cJPT4taWNSaTLS64Dp67wuXigPZ &
-IDXR_PID=$(echo $!)
+IDXR_PID=$(pgrep mina-indexer)
```
Comment (jenr24G) [inline test:104]: regression?

Comment (trevorbernard) [reply test:104]: `echo $!` was grabbing the wrong PID because of the subshell. So I reverted my suggestion

Comment (jenr24G) [reply test:104]: I'm seeing the same thing

**Review by Isaac-DeFrain — DISMISSED** [review]
LGTM

**Review by Isaac-DeFrain — APPROVED** [review]

**Pull request merged**